Assets/MRTK/Tests/PlayModeTests/PointerBehaviorTests.cs

```diff
@@ -413,6 +413,7 @@ public IEnumerator TestTeleportLayers()
             yield return new WaitForSeconds(1.0f / iss.InputSimulationProfile.HandGestureAnimationSpeed + 0.1f);
 
             TeleportPointer teleportPointer = rightHand.GetPointer<TeleportPointer>();
+            teleportPointer.PrioritizedLayerMasksOverride = new LayerMask[1] { UnityEngine.Physics.AllLayers };
```

Comment:
This seems weird. Why do we have to override this?

Reply:
Our original unit tests made it so the `Ignore Raycasts` layermask was actually getting hit by raycasts. This is just some realignment work to make the tests work as expected.